implementations/skint/0/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# -*- mode: dockerfile; coding: utf-8 -*-
+FROM debian:bookworm-slim AS build
+RUN apt-get update && apt-get -y --no-install-recommends install \
+      build-essential ca-certificates git \
```

Comment:
`ca-certificates git` should be unnecessary for building the release container. Does the build work if you don't install them?

Reply:
Good catch, thanks! It does not need them. I copy pasted some lines so propably ended there that way.